components/renku_data_services/platform/orm.py

```diff
@@ -50,3 +52,42 @@ def dump(self) -> models.PlatformConfig:
             creation_date=self.creation_date,
             updated_at=self.updated_at,
         )
+
+
+class UrlRedirectsORM(BaseORM):
+    """The url redirects."""
+
+    __tablename__ = "url_redirects"
+
+    id: Mapped[ULID] = mapped_column("id", ULIDType, primary_key=True, default_factory=lambda: str(ULID()), init=False)
```

Comment:
suggestion: it is even better if you add the server-side default for the id so that postgres will just make a ulid. It helps if you do mgirations or other changes. You will need to import `text` from `sqlalchemy` too at the top.

```suggestion
    id: Mapped[ULID] = mapped_column("id", ULIDType, primary_key=True, default_factory=lambda: str(ULID()), init=False, server_default=text("generate_ulid()"))
```